src/main/java/com/springbootpractice/examples/employeemanagement/controller/EmployeeController.java

```diff
@@ -0,0 +1,51 @@
+package com.springbootpractice.examples.employeemanagement.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.springbootpractice.examples.employeemanagement.data.Employee;
+import com.springbootpractice.examples.employeemanagement.service.EmployeeService;
+
+@RestController
+public class EmployeeController {
+
+	@Autowired
+	private EmployeeService employeeService;
+
+	@GetMapping("/getallemployees")
```

Comment:
When you create endpoints related to same resource, you should have the noun as the mapping with GET and POST differentiating what they do. For eg if we create employees, the requests should be

/employee (POST) to create
/employee/{id} (PUT) to update
/employee/{id} (GET) to get an employee
/employees (GET) list all employees
/employee/{id} (DELETE) delete an employee